index.d.ts

```diff
@@ -0,0 +1,456 @@
+declare module 'oidc-provider' {
+    import { Request, Response, Handler } from 'express';
+    import Koa = require('koa');
+    import Router = require('koa-router');
+    import { IncomingMessage, ServerResponse } from 'http';
+    import EventEmitter = require('events');
+    import { JWK } from 'node-jose';
+
+    type SubjectType = 'public' | 'parwise';
+
+    interface IClient {
+        applicationType?: 'web' | 'native';
```

Comment:
is there syntax to specify the default value? if so `web` is default.

Reply:
No, interfaces cannot have default values as I understand it. We could perhaps convert it to a class instead?